### ISSUE TITLE
How to get the output in mnist_siamese_graph example

### ISSUE THREAD

**Issue by longma307** [fchollet/keras#4348]
def create_base_network(input_dim):
    '''Base network to be shared (eq. to feature extraction).
    '''
    seq = Sequential()
    seq.add(Dense(128, input_shape=(input_dim,), activation='relu'))
    seq.add(Dropout(0.1))
    seq.add(Dense(128, activation='relu'))
    seq.add(Dropout(0.1))
    seq.add(Dense(128, activation='relu'))
    return seq

This function is supposed to feature extraction. 

After model.fit(), how can I get the output given input, e.g. x1 (one input of the Siamese network)

**Comment by dhruvilbadani** [comment]
@longma307 Hi! How did you solve this? Thanks!
